pkg/declarativeconfig/access_scope.go

```diff
@@ -0,0 +1,63 @@
+package declarativeconfig
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"gopkg.in/yaml.v3"
+)
+
+// AccessScope is representation of storage.AccessScope that supports transformation from YAML.
+type AccessScope struct {
+	Name        string `yaml:"name,omitempty"`
+	Description string `yaml:"description,omitempty"`
```

Comment:
Not for this PR, more or less a generic question:
I wonder if it makes sense, seeing as this could grow in amount of resources and almost all resources seem to have `name & description` to create a separete `Metadata` struct and embed this within here, i.e.:
```go
type Metadata struct {
	Name        string `yaml:"name,omitempty"`
	Description string `yaml:"description,omitempty"`
}

type AccessScope {
          Metadata
          Rules Rules `yaml:"rules,omitempty"`
}
```

Reply:
Yep, I will create a follow-up JIRA